subsys/usb/usb_device.c

```diff
@@ -189,24 +193,32 @@ static void usb_print_setup(struct usb_setup_packet *setup)
 static bool usb_handle_request(struct usb_setup_packet *setup,
 			       int32_t *len, uint8_t **data)
 {
-	uint32_t type = REQTYPE_GET_TYPE(setup->bmRequestType);
-	usb_request_handler handler = usb_dev.req_handlers[type];
-
-	if (type >= MAX_NUM_REQ_HANDLERS) {
-		LOG_DBG("Error Incorrect iType %d", type);
-		return false;
-	}
-
-	if (handler == NULL) {
-		LOG_DBG("No handler for reqtype %d", type);
+	switch (setup->RequestType.type) {
+	case  REQTYPE_TYPE_STANDARD:
```

Comment:
nit: change the double space to a single space.

Reply:
thanks, will fix (@nzmichaelh Please "request changes" for this kind of issues)

Reply:
fixed